openslides_backend/action/actions/user/import_.py

```diff
@@ -0,0 +1,108 @@
+from typing import Any, Dict, List
+
+from ....models.models import ActionWorker
+from ....permissions.management_levels import OrganizationManagementLevel
+from ....shared.exceptions import ActionException
+from ....shared.patterns import fqid_from_collection_and_id
+from ....shared.schema import required_id_schema
+from ...mixins.import_mixins import ImportMixin, ImportStatus
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from .create import UserCreate
+from .update import UserUpdate
+from .user_mixin import DuplicateCheckMixin
+
+
+@register_action("user.import")
+class UserImport(DuplicateCheckMixin, ImportMixin):
+    """
+    Action to import a result from the action_worker.
+    """
+
+    model = ActionWorker()
+    schema = DefaultSchema(ActionWorker()).get_default_schema(
+        additional_required_fields={
+            "id": required_id_schema,
+            "import": {"type": "boolean"},
+        }
+    )
+    permission = OrganizationManagementLevel.CAN_MANAGE_USERS
+    skip_archived_meeting_check = True
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        store_id = instance["id"]
+        worker = self.datastore.get(
+            fqid_from_collection_and_id("action_worker", store_id), ["result"]
+        )
+        if (worker.get("result") or {}).get("import") != "account":
+            raise ActionException("Wrong id doesn't point on account import data.")
+
+        # handle abort in on_success
+        if not instance["import"]:
+            return {}
```

Comment:
You can move this up to return before fetching anything

Reply:
No, we must fetch the action worker to see, if this is a account import action worker. Else we have a way to remove
every action_worker.

Reply:
right